vcd/resource_vcd_tm_content_library_item.go

```diff
@@ -0,0 +1,238 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v3/govcd"
+	"github.com/vmware/go-vcloud-director/v3/types/v56"
+)
+
+const labelTmContentLibraryItem = "Content Library Item"
+
+func resourceVcdTmContentLibraryItem() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVcdTmContentLibraryItemCreate,
+		ReadContext:   resourceVcdTmContentLibraryItemRead,
+		// TODO: TM: Update not supported yet
+		// UpdateContext: resourceVcdTmContentLibraryItemUpdate,
+		DeleteContext: resourceVcdTmContentLibraryItemDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcdTmContentLibraryItemImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true, // TODO: TM: Update not supported yet
+				Description: fmt.Sprintf("Name of the %s", labelTmContentLibraryItem),
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true, // TODO: TM: Update not supported yet
+				Description: fmt.Sprintf("The description of the %s", labelTmContentLibraryItem),
+			},
+			"content_library_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true, // TODO: TM: Update not supported yet
```

Comment:
NIT. About the comment - technically, I don't think this will support update anyway

Reply:
Removed